server/routes/visit.test.ts

```diff
@@ -1654,153 +1654,6 @@ describe('/visit/:reference/update/select-date-and-time', () => {
         })
     })
   })
-
-  describe.skip('POST /visit/:reference/update/select-date-and-time', () => {
```

Comment:
Removing these blocks of tests (they were 'skipped' anyway) for now but will add coverage for update journey separately, likely as part of VB-1204.